packages/docs/SUMMARY.md

```diff
@@ -17,6 +17,13 @@
   - [Getting an Account and Funds](getting-started/faucet.md)
 - [Glossary](GLOSSARY.md)
 
+## Celo Gold Holder Guide
+- [Quick Start](celo-gold-holders/quick-start.md)
+- [Using a Ledger Wallet](celo-gold-holders/ledger.md)
+- [Understanding ReleaseGold](celo-gold-holders/release-gold.md)
+- [Voting for Validator Groups](celo-gold-holders/release-gold.md)
+- [Voting for Governance](celo-gold-holders/governance.md)
```

Comment:
Suggest we rename file so that the URL is clearer that this involves voting e.g. celo-gold-holders/vote-governance.md (also avoids confusing with other governance-related docs/files)